webapp/src/app/ui/avatar/avatar.component.ts

```diff
@@ -0,0 +1,52 @@
+import { ChangeDetectionStrategy, Component, computed, input, signal } from '@angular/core';
+import type { ClassValue } from 'clsx';
+import type { VariantProps } from 'class-variance-authority';
+import { cn } from 'app/utils';
+import { cva } from 'app/storybook.helper';
+import { NgOptimizedImage } from '@angular/common';
+
+const [avatarVariants, args, argTypes] = cva('relative flex shrink-0 overflow-hidden rounded-full', {
+  variants: {
+    size: {
+      default: 'h-10 w-10',
+      sm: 'h-6 w-6 text-xs',
+      lg: 'h-14 w-14 text-lg'
+    }
+  },
+  defaultVariants: {
+    size: 'default'
+  }
+});
+
+export { args, argTypes };
+
+interface AvatarVariants extends VariantProps<typeof avatarVariants> {}
+
+@Component({
+  selector: 'app-avatar',
+  standalone: true,
+  imports: [NgOptimizedImage],
+  changeDetection: ChangeDetectionStrategy.OnPush,
```

Comment:
Yes, you are correct, I just removed it. From what I've read in the Angular docs it can improve performance when using ng-templates (which I tested out in an earlier version).